client/me/profile-gravatar/index.jsx

```diff
@@ -3,17 +3,19 @@
  */
 import React from 'react';
 import debugFactory from 'debug';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import Animate from 'components/animate';
 import Gravatar from 'components/gravatar';
 import eventRecorder from 'me/event-recorder';
+import { isEnabled } from 'config';
 
 const debug = debugFactory( 'calypso:me:sidebar-gravatar' );
 
-export default React.createClass( {
+const ProfileGravatar = React.createClass( {
```

Comment:
Agreed. The code will either be refactored in a future tracking PR for this feature, or will be removed entirely when the feature launches.